data/geometry/DEER_Asm.osm

```diff
@@ -2612,6 +2612,6 @@ OS:SpaceType,
   ,                                       !- Default Schedule Set Name
   ,                                       !- Group Rendering Name
   ,                                       !- Design Specification Outdoor Air Object Name
-  ,                                       !- Standards Building Type
+  Asm,                                    !- Standards Building Type
```

Comment:
need to add Standards Building Type to the Plenum Standards Space Type in DEER models for the lookup mapping to function